### ISSUE TITLE
Mädchennamen (once again...)

### ISSUE THREAD

**Issue by sekolb** [Joachim-Raff-Archiv/portal-app#34]
Momentan steht in der Briefliste bei den Briefen von 1851 etc. als Briefempfängerin "Doris Raff", was ja sehr falsch ist, da sie erst 1859 diesen Namen annimmt. Kann man da was machen?

**Comment by riedde** [comment]
Ja, wenn wir den Namen ein Datum zuordnen, ab wann dieser Name verwendet wird, dann können wir das im Portal auch entsprechend anzeigen. Ich habe gerade recherchiert und ich glaube, dass es besser ist, wenn wir die Angabe von Namen generell ändern. Können wir uns das für das nächste Update vornehmen? Ich befürchte, dass wir d eine größere Aktion.

**Comment by sekolb** [comment]
Klar, machen wir doch! Manchmal dürfte das ja auch ganz schön schwierig sein. Dann müsste man so etwas machen können wie "not before", "not after".

**Comment by riedde** [comment]
Genau, und dann können wir nämlich auch die Zuordnung des Adelsprädikats (von Bülow) vom Rest abtrennen.

**Comment by sekolb** [comment]
Sehr schön! :)

**Comment by sekolb** [comment]
Hast du hier mit Henze eine Lösung gefunden, die nicht allzu aufwendig ist? Ich habe jetzt in meiner Diss alle Briefe an Doris Genast ins Portal verlinkt - es wäre etwas seltsam, wenn dort immer noch Doris Raff angegeben ist... :)
